plugins/filter_ecs/ecs.c

```diff
@@ -1694,6 +1694,15 @@ static struct flb_config_map config_map[] = {
      "Defaults to 51678"
     },
 
+    {
+     FLB_CONFIG_MAP_INT, "agent_endpoint_retries", FLB_ECS_FILTER_METADATA_RETRIES,
```

Comment:
`config_map` entry `def_value` must be a string
https://github.com/fluent/fluent-bit/blob/eceeb5fe7b550a6a6069b79850b635f985f5ac0a/include/fluent-bit/flb_config_map.h#L69

See NightFall for an example
https://github.com/fluent/fluent-bit/blob/d47713ea0ab50fd757775e43958233a09460db9d/plugins/filter_nightfall/nightfall.c#L562

The string value may be autoconverted to one of the FLB_CONFIG_MAP_INT type here:
https://github.com/fluent/fluent-bit/blob/1532471353681da1ed86c2945e2b2879904465b0/src/flb_config_map.c#L271-L273

Reply:
Suggested edit: Revise FLB_ECS_FILTER_METADATA_RETRIES from 2 to "2"

https://github.com/fluent/fluent-bit/blob/e207bbadea01d3cfcd2e35f48aec28142c2420aa/plugins/filter_ecs/ecs.h#L33

Reply:
thanks